src/com/company/model/DatabaseManager.java

```diff
@@ -0,0 +1,63 @@
+package com.company.model;
+
+import com.company.view.ConsoleReader;
+
+import java.sql.*;
+
+/**
+ * Created by yulia on 28.09.16.
+ */
+public class DatabaseManager {
+    public static Connection connection;
+    private static Statement stmt;
+
+    public static Connection connect(String database, String user, String password) throws SQLException, ClassNotFoundException {
+
+            try {
+                Class.forName("org.postgresql.Driver");
+            } catch (ClassNotFoundException e) {
+                throw new RuntimeException("Please add jdbc jar to project.", e);
+            }
+            try {
+                connection = DriverManager.getConnection(
+                        "jdbc:postgresql://localhost:5432/" + database, user,
+                        password);
+            } catch (SQLException e) {
+                throw new RuntimeException(
+                        String.format("Can't get connection for model:%s user:%s",
+                                database, user),
+                        e);
+
+            }
+            System.out.println("Connection successful");
+
+        stmt = connection.createStatement();
+        return connection;
+    }
+
+    public Statement getStatement(){
+        return stmt;
+    }
+
+    public static Connection getConnection()  {
+        if(connection == null) {
```

Comment:
this check means your need to apply Singleton pattern to this class. So it will have only one method - **getConnection**(...) so **connect**(...) will be private and will be called only inside this method.